### PR TITLE
feat(chat-e2e): publish single chat tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

publish single chat tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat-e2e/src/assertions/conversationToApproveAssertion.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { PublishEntityAssertion } from '@/src/assertions/publishEntityAssertion';
+import { ConversationsToApproveTree } from '@/src/ui/webElements/entityTree';
+
+export class ConversationToApproveAssertion extends PublishEntityAssertion<ConversationsToApproveTree> {}
```
Comment (nartovm) [inline apps/chat-e2e/src/assertions/conversationToApproveAssertion.ts:4]: Is it a template for future improvements or something like that?

File: apps/chat-e2e/src/assertions/organizationConversationAssertion.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { SideBarEntityAssertion } from '@/src/assertions/sideBarEntityAssertion';
+import { OrganizationConversationsTree } from '@/src/ui/webElements/entityTree';
+
+export class OrganizationConversationAssertion extends SideBarEntityAssertion<OrganizationConversationsTree> {}
```
Comment (nartovm) [inline apps/chat-e2e/src/assertions/organizationConversationAssertion.ts]: Is it a template for future improvements or something like that?

File: apps/chat-e2e/src/assertions/sideBarEntityAssertion.ts [hunk 3]
```diff
@@ -9,11 +9,14 @@ import { SideBarEntitiesTree } from '@/src/ui/webElements/entityTree/sidebar/sid
 import { ThemesUtil } from '@/src/utils/themesUtil';
 import { expect } from '@playwright/test';
 
-export class SideBarEntityAssertion<T extends SideBarEntitiesTree> {
+export class SideBarEntityAssertion<
```
Comment (nartovm) [inline apps/chat-e2e/src/assertions/sideBarEntityAssertion.ts:12]: Probably we should consider moving out methods like assertEntityArrowIconColor and assertEntityArrowIconState to the EntityTreeAssertion class

**Review by nartovm — DISMISSED** [review]

File: apps/chat-e2e/src/assertions/conversationToApproveAssertion.ts (hunk 1 above)
Comment (irinakartun) [reply apps/chat-e2e/src/assertions/conversationToApproveAssertion.ts:4]: this class was developed for specific assertions associated with the "Approval Required" section which will be needed in the future:
image

File: apps/chat-e2e/src/assertions/organizationConversationAssertion.ts (hunk 2 above)
Comment (irinakartun) [reply apps/chat-e2e/src/assertions/organizationConversationAssertion.ts]: moved instantiation of `organizationConversationAssertion` class directly to fixtures

File: apps/chat-e2e/src/assertions/sideBarEntityAssertion.ts (hunk 3 above)
Comment (irinakartun) [reply apps/chat-e2e/src/assertions/sideBarEntityAssertion.ts:12]: agreed, fixed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by nartovm — APPROVED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2182.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**